br-webui/views/camera.liquid

```diff
@@ -482,6 +540,129 @@
 	
 	socket.emit('get v4l2 cameras'); // get v4l2 options
 	
+	//PATRICK IS WORKING
+	// Deal with stream types
+	/// Populate valid video stream types
+	function populateStreamTypeSelect() {
+		// Get combobox and add all options, the value of each option is the string pipeline
+		var streamTypeSelect = document.getElementById("streamTypeSelect")
```

Comment:
const?

Reply:
There is an `appendChild` in the following block.

Reply:
It appears that js const are not const.
![image](https://user-images.githubusercontent.com/1215497/86025996-472dad00-ba05-11ea-8597-765c00de7e79.png)